### PR TITLE
Run foreman tasks under foreman user

### PR DESCRIPTION
**Pull request by ares**
Also remove redundant template in wrong directory

### CODE REVIEW

**Comment by ares** [comment]
@inecas could you please test on that affected machine and then ACK?

File: modules/foreman/manifests/config.pp [hunk 1]
```diff
@@ -83,12 +83,21 @@
...
     environment => ["RAILS_ENV=${foreman::environment}", "BUNDLER_EXT_NOSTRICT=1"],
     command     => "/usr/bin/${katello::params::scl_prefix}rake security:generate_token",
     path        => "/bin:/usr/bin",
-    creates     => "${foreman::app_root}/config/initializers/local_secret_token.rb",
+    creates     => $foreman_token_file,
+    user        => $foreman::user,
+  } ~>
+
+  file {
+    "${foreman_token_file}":
```
Comment (iNecas) [inline modules/foreman/manifests/config.pp]: Everywhere is used one line style, lets keep the consistency:

```
file { "${foreman_token_file}" :
```

**Comment by xsuchy** [comment]
ACK

File: modules/foreman/manifests/config.pp (hunk 1 above)
Comment (ares) [inline modules/foreman/manifests/config.pp]: Done (I didn't use spacebars exactly as you suggested, other code seems not to use them as well)

**Comment by iNecas** [comment]
ACK. For those who interested: we now don't have to redirect the outputs from rake commands somewhere as they go by default into main.log. In this case there was a problem with permissions.

**Pull request merged**